linkedList.js

```diff
@@ -3,6 +3,108 @@
  * @description  Main Module
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
-function linkedListGenerator(){
+function linkedListGenerator() {
+  let head = null;
+  let tail = null;
+
+  function getHead() {
+    return head;
+  }
+
+  function getTail() {
+    if (tail === null) {
+      return null;
+    };
+
+    tail = head;
+    while (tail.next !== null) {
+      tail = tail.next;
+    }
+    return tail;
+  }
+
+  function add(value) {
+    let newNode = {};
+    newNode.value = value;
+    newNode.next = null;
+
+    if (head === null) {
+      newNode.next = head;
+      head = newNode;
+    } else {
+      let endOfNodeList = head;
+      while (endOfNodeList.next !== null) {
+        endOfNodeList = endOfNodeList.next;
+      }
+      endOfNodeList.next = newNode;
+    };
+
+    tail = newNode;
+    return newNode;
+  }
+
+  function get(number) {
+
+    let currentNode = head;
+    let counter = 0;
+
+    while (currentNode) {
+      if (counter === number) {
+        return currentNode;
+      }
+      counter++;
+      currentNode = currentNode.next;
+    }
+    return false;
+  }
+
+  function remove(number) {
+
+    if (number === 0) {
```

Comment:
Although there's no test for this, you should handle deleting the node at position 0 with an empty linked list.